frontend/www/js/omegaup/contest/edit.ts

```diff
@@ -467,8 +474,8 @@ OmegaUp.on('ready', () => {
           'accept-request': ({ username }: { username: string }) => {
             this.arbitrateRequest(username, true);
           },
-          'deny-request': ({ username }: { username: string }) => {
-            this.arbitrateRequest(username, false);
+          'deny-request': (username: string, resolutionText: string = '') => {
```

Comment:
Here you can use object deestructuring:
```suggestion
          'deny-request': ({
            username,
            resolutionText,
          }: {
            username: string;
            resolutionText: null | string;
          }) => {
```